resources/test_fonts/README.md

```diff
@@ -1,6 +1,15 @@
-The ttx files are the masters. To update the binaries:
+# test files
+
+This directory contains files used for testing. The masters are the ttx files;
+these are human readable/editable. From these, we generate the binary ttf
+files that are the actual test inputs.
+
+## rebuilding
+To update the binaries, run the `./resources/test_fonts/rebuild.sh` from the
+repo root. This script will install the correct version of fonttools, and then
```

Comment:
It's a papercut that you have to run from root, should work regardless of where I run it from

Reply:
totally agree, but is there an easy way for me to resolve all the paths I need to access, if I'm starting in an arbitrary working directory? I don't do much shell scripting.

Reply:
```bash
SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
```

I'm no shell scripter either but according to stack overflow that's how you get a bash script's directory. You could then either prefix that to all relative paths, or cd or pushd to it and popd at the end

https://stackoverflow.com/a/246128

Reply:
okay thank you for doing my googling for me, that ended up being a nice bit of cleanup. :)